script/slurm/slurm_nss_matrix.sh

```
@@ -112,16 +112,15 @@ if [[ -n "${NSS_VERSION:-}" ]]; then
     uv venv --python 3.11 "${PYPI_VENV}"
     source "${PYPI_VENV}/bin/activate"
-    uv pip install "nemo-safe-synthesizer[cu129,engine]==${NSS_VERSION}" \
+    uv pip install "nemo-safe-synthesizer[cu129]==${NSS_VERSION}" \
         --index https://flashinfer.ai/whl/cu129 \
         --index https://download.pytorch.org/whl/cu129 \
-        --index https://wheels.vllm.ai/88d34c6409e9fb3c7b8ca0c04756f061d2099eb1/cu129 \
         --index-strategy unsafe-best-match
     NSS_RUN_CMD="${PYPI_VENV}/bin/safe-synthesizer"
     echo "[NSS SLURM] Using PyPI install: nemo-safe-synthesizer==${NSS_VERSION}"
 else
     source "${NSS_DIR}/.venv/bin/activate"
```

Shellcheck (0.11.0)

[info] Not following: ./.venv/bin/activate was not specified as input (see shellcheck -x).

(SC1091)